styles.css

```diff
@@ -678,4 +678,549 @@ main hr {
+  }
+
+  .img-placeholder-desktop {
+    flex-basis: 56%;
+    width: 45vw;
+    height: 31vw;
+    display: flex;
+    align-items: center;
+    margin-right: 1.6vw;
+    max-width: 1445px;
+    max-height: 560px;
+  }
+
+  .img-daily-desktop {
+    width: 100%;
+    height: 100%;
+  }
+
+  .daily-selection article {
+    flex-basis: 44%;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    margin-top: 0;
+    align-items: flex-start;
+    gap: 1.92%;
+  }
+
+  .daily-selection article h3 {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .daily-selection p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+  }
+
+  .tags {
+    width: 1.57vw;
+    height: 2.7vw;
+    gap: 20px;
+  }
+
+  #desktop-ul li {
+    padding-right: 0.83vw;
+    padding-left: 0.83vw;
+    padding-bottom: 0.69vw;
+    padding-top: 0.69vw;
+  }
+
+  .see-more-container {
+    width: 22.7vw;
+    margin-bottom: 24px;
+  }
+
+  .see-more-btn {
+    width: 8.6vw;
+    height: 3.3vw;
+  }
+
+  /* Here daily selection ends */
+
+  /* Here starts project grid */
+
+  .projects-grid {
+    display: none;
+  }
+
+  .projects-grid-desktop {
+    display: grid;
+    width: 80.2vw;
+    gap: 1.6vw;
+    grid-template-columns: 31.94% 31.94% 31.94%;
+    margin-bottom: 120px;
+    margin-top: 1.6vw;
+    max-width: 1445px;
+    min-width: 1040px;
+    min-height: 440px;
+  }
+
+  .project-card {
+    height: 31.1vw;
+    margin-bottom: 24px;
+    align-items: baseline;
+  }
+
+  .card-one {
+    background: url(vectors/Img_desktop1.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    border: 2px solid black;
+  }
+
+  .card-two {
+    background: url(vectors/Img_desktop2.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .card-three {
+    background: url(vectors/Img_desktop3.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .card-four {
+    background: url(vectors/Img_desktop4.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .card-five {
+    background: url(vectors/Img_desktop2.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .card-six {
+    background: url(vectors/Img_desktop3.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .card-link {
+    color: #fff;
+  }
+
+  .card-title {
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    width: 23.4vw;
+    margin-top: 12.36vw;
+    margin-left: 1.1vw;
+    margin-right: 1.1vw;
+  }
+
+  .card-description {
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 24px;
+    margin-top: 1.6vw;
+    width: 23.4vw;
+    margin-left: 1.1vw;
+    margin-right: 1.1vw;
+  }
+
+  .card-tags {
+    margin-top: 1.6vw;
+    margin-bottom: 1.11vw;
+    height: 40px;
+    align-items: center;
+    margin-left: 1.1vw;
+    margin-right: 1.1vw;
+  }
```

Comment:
- [ ] Kindly adjust the paddings here just as you did to the other one
![image](https://user-images.githubusercontent.com/62705947/186943675-f830bf89-0b83-45db-9ad0-f1ca45784196.png)